client/Containers/PaginationControl.jsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import styled from 'styled-components';
+import productPagination, {
+  setProductPagination,
+  nextPage,
+  prevPage,
+  goToPage,
+  sizePage,
+  sortPage,
+} from '../store/product/productPagination';
+import buildProductQuery from '../components/utils/buildProductQuery';
+import { fetchProductList } from '../store/product/productList';
+
+// const mapStateToProps = ({ productPagination }) => ({ productPagination });
+
+// const mapDispatchToProps = (dispatch) => ({
+//   setPagination: (productPagination) =>
+//     dispatch(setProductPagination(productPagination)),
+//   paginationFuncs: {
+//     nextPage: () => dispatch(nextPage()),
+//     prevPage: () => dispatch(prevPage()),
+//     goToPage: (page) => dispatch(goToPage(page)),
+//     sizePage: (size) => {
+//       dispatch(sizePage(size));
+//       dispatch(goToPage(1));
+//     },
+//     sortPage: (sort) => dispatch(sortPage(sort)),
+//   },
+// });
```

Comment:
Like `FilterSortControl` had to use hooks to easily grab history for changing the URL